apps/a11y-tests/src/axe-sarif-converter/axe-utils.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+/// <reference path="./axe-extension.ts" />
```

Comment:
You shouldn't need this if you get the custom typings in the right place and typeRoots set up.

Reply:
Code in `apps/a11y-tests/src/axe-sarif-converter` is a temporary copy and will be removed once official converter is published. In the meantime I will try to make it compile against OUFR's tslint/tsc options, but would prefer not to make re-organizing/refactoring changes.